client/src/controllers/LocaleController.ts

```diff
@@ -58,7 +59,10 @@ export class LocaleController extends Controller<HTMLSelectElement> {
       if (!timeZone) return;
       const localized = LocaleController.getTZLabel(timeZone);
       const option = opt;
-      option.textContent = `${option.textContent}: ${localized}`;
+      const template = gettext('%s: %s');
```

Comment:
I know this is in JS and you're just trying to get a generic separator (with a colon) but we should try to use the same approach as documented here. https://docs.wagtail.org/en/stable/contributing/translations.html#translations-within-python

Here's a suggestion, maybe it's too verbose, but this should also help translators.

```suggestion
      const template = gettext('%(time_zone_option)s: %(localized_time_zone_label)s');
```

This should hopefully help your replacement on the next line be able to be more robust.